DataConnectors/illusive Attack Management System.json

```diff
@@ -0,0 +1,108 @@
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "CommonSecurityLog",
+            "baseQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"illusive\"\n| where DeviceProduct == \"illusive\"\n"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Number of Incidents in in the last 30 days in which Trigger Type is found",
+            "query": "union CommonSecurityLog | where (DeviceEventClassID == \"illusive:login\" or DeviceEventClassID == \"illusive:access\" or DeviceEventClassID == \"illusive:suspicious\")| where Message !contains \"hasForensics\" | where TimeGenerated > ago(30d) | summarize by DestinationServiceName ,DeviceCustomNumber2 | summarize incident_count=count() by DestinationServiceName"
+        },
+		{
+            "description" : "Top 10 alerting hosts in the last 30 days",
+            "query": "union CommonSecurityLog | where (DeviceEventClassID == \"illusive:login\" or DeviceEventClassID == \"illusive:access\" or DeviceEventClassID == \"illusive:suspicious\")| where Message !contains \"hasForensics\" | where TimeGenerated > ago(30d) | summarize by AlertingHost=iff(SourceHostName != \"\" and SourceHostName != \"Failed to obtain\", SourceHostName, SourceIP)   ,DeviceCustomNumber2 | where AlertingHost != \"\" and AlertingHost != \"Failed to obtain\" | summarize incident_count=count() by AlertingHost | order by incident_count | limit 10"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog (illusive)",
+            "lastDataReceivedQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"illusive\"\n| where DeviceProduct == \"illusive\"\n\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "\nCommonSecurityLog\n| where DeviceVendor == \"illusive\"\n| where DeviceProduct == \"illusive\"\n\n | summarize LastLogReceived = max(TimeGenerated)\n | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "read": true,
+                    "write": true,
+                    "delete": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
```

Comment:
Please add a link to Illusive's website for information on Illusive and what data they'll see.

Reply:
Thank you @preetikr 
Those are the instructions steps how to connect the our connector. We don't have any additional info in our website on the data we send via CEF (or how to use it). Do you expect just general link to our website or something else?

Reply:
Yes, a general link would be great. Thanks.

Reply:
Yes, a general link would be great. Thanks.

Reply:
Fixed